admin/app/javascript/spree/admin/controllers/variants_form_controller.js

```diff
@@ -1052,4 +1061,14 @@ export default class extends CheckboxSelectAll {
       }
     }
   }
+
+  toggleInventoryForm(value) {
+    if (!this.inventoryFormTarget) return
+
+    if (value == 'show') {
```

Comment:
`value` can be a `boolean` instead of `string`
```suggestion
    if (value) {
```